spec.md

```diff
@@ -427,22 +427,28 @@ how Platforms might expose these values to their users.
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| create | [InputParametersSchema](#input-parameters-schema-object) | The schema definition for creating a Service Instance. |
-| update | [InputParametersSchema](#input-parameters-schema-object) | The schema definition for updating a Service Instance. |
+| create | [RequestResponseSchema](#request-response-schema-object) | The Service Broker schema definition for creating a Service Instance and the associated response. |
```

Comment:
I don't think the type should change here - the response part doesn't seem like it has meaning for creating a service instance.

Reply:
I think there is value in allowing the JSON Schema requests for all responses, this allows a broker to define what it means (give some context text to the UI?) for it's response for any OSBAPI defined fields at the very least.